### PR TITLE
🐛 fix: include alertmanager path prefix in vmalert config

### PR DESCRIPTION
**Pull request by mhkarimi1383**

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/templates/_helpers.tpl [hunk 1]
```diff
@@ -127,7 +127,7 @@ notifierConfigRef:
 {{- $alertManagerReplicas := .Values.alertmanager.spec.replicaCount | default 1 }}
 notifiers:
     {{- range $n := until (int $alertManagerReplicas) }}
-    - url: {{ printf "http://%s-%s-%d.%s-%s.%s.svc:9093" "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $n "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $.Release.Namespace }}
+    - url: {{ printf "http://%s-%s-%d.%s-%s.%s.svc:9093%s" "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $n "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $.Release.Namespace (.Values.alertmanager.spec.routePrefix) }}
```
Comment (AndrewChubatiuk) [inline charts/victoria-metrics-k8s-stack/templates/_helpers.tpl:130]:
```suggestion
    - url: {{ printf "http://%s-%s-%d.%s-%s.%s.svc:9093%s" "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $n "vmalertmanager" (include "victoria-metrics-k8s-stack.fullname" $) $.Release.Namespace ($.Values.alertmanager.spec.routePrefix) }}
```

Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/templates/_helpers.tpl:130]: Could you please an entry to a changelog as well?

Comment (mhkarimi1383) [reply charts/victoria-metrics-k8s-stack/templates/_helpers.tpl]: Will do it.

**Review by AndrewChubatiuk — APPROVED** [review]
LGTM

**Pull request merged**